### PR TITLE
Testing Hooks with `ps_qualityassurance`

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Testing Hooks with `ps_qualityassurance`
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: 
| Fixed issue or discussion?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

File: content/project-organization/quality-council/processes/test-hooks-with-ps_qualityassurance/_index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+* For the hook name, use `actionProductPriceCalculation`
+* For the content, use 
+```php
+  if ($params['use_tax']) {
+      $params['price'] = 42.00;
+  } else {
+      $params['price'] = 35.00;
+  }
+```
+* Save
```
Comment (florine2623) [inline content/project-organization/quality-council/processes/test-hooks-with-ps_qualityassurance/_index.md:53]: Could you change Save to Register ?
Screenshot 2024-03-27 at 10 35 03

File: content/project-organization/quality-council/processes/test-hooks-with-ps_qualityassurance/_index.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+#### Example
+
+* Go the configuration page of the module
+* Click on the tab "Register Hook"
+* For the hook name, use `displayProductListReviews`
+* For the content, use 
+```php
+  return 'Your product "' . $params['product']['name'] . '" has the ID ' . $params['product']['id'] . '.';
+```
+* Save
```
Comment (florine2623) [inline content/project-organization/quality-council/processes/test-hooks-with-ps_qualityassurance/_index.md:80]: Same as above, could you change Save to Register as it is in the module ?
Screenshot 2024-03-27 at 10 35 03

File: content/project-organization/quality-council/processes/test-hooks-with-ps_qualityassurance/_index.md (hunk 1 above)
Comment (Progi1984) [reply content/project-organization/quality-council/processes/test-hooks-with-ps_qualityassurance/_index.md]: Fixed

File: content/project-organization/quality-council/processes/test-hooks-with-ps_qualityassurance/_index.md (hunk 2 above)
Comment (Progi1984) [reply content/project-organization/quality-council/processes/test-hooks-with-ps_qualityassurance/_index.md]: Fixed

**Review by florine2623 — APPROVED** [review]
Thanks @Progi1984 LGTM ✅

**Pull request merged**